zerver/models/users.py

```diff
@@ -844,7 +844,7 @@ def can_create_web_public_streams(self) -> bool:
     def can_manage_default_streams(self) -> bool:
         return self.is_realm_admin
 
-    def can_subscribe_other_users(self) -> bool:
+    def can_subscribe_others_to_all_streams(self) -> bool:
         return self.has_permission("can_add_subscribers_group")
```

Comment:
Maybe call this `can_subscribe_others_to_all_accessible_streams` just to make super clear that this function does not attempt to check basic access.